packages/amplify-category-auth/provider-utils/awscloudformation/index.js

```diff
@@ -607,6 +723,145 @@ function removeDeprecatedProps(props) {
   return props;
 }
 
+async function addAdminAuth(context, authResourceName, operation, adminGroup, functionName) {
```

Comment:
We don't have to do this right now, but we should start breaking up this index file.